.github/workflows/verify.yml

```diff
@@ -0,0 +1,69 @@
+# this workflow contains fmt,build,ut fot tikv-client
+name: verify
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  fmt:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+          cache: maven
+      - name: fmt
+        run: |
+          export LC_ALL=en_US.UTF-8
+          export LANG=en_US.UTF-8
+          export LANGUAGE=en_US.UTF-8
+          mvn mvn-scalafmt_2.12:format -Dscalafmt.skip=false -B
```

Comment:
I've tested this branch but this command is not able to succeed. It reports error `Config path is null or empty`. According to https://github.com/SimonJPegg/mvn_scalafmt#minimal-working-pom-xml, there shall be a config.

Reply:
The config is already in the pom.xml before. Besides the error seems have something to do with the error path you excuted the command , maybe you can try to excute this command under the 'tispark' directory

Reply:
At tispark directory. It's not the parent POM, but `db-random-test`.

Reply:
i find that maven 3.6.3 lead to the error, try other maven version can be effective